docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/ExtractPixels.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.IO;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ExtractPixelsExample
+    {
+        public static void ExtractPixels()
+        {
+            var mlContext = new MLContext();
+
+            // Downloading a few images, and an images.tsv file, that contains a list of the files, from the dotnet/machinelearning/test/data/images/
+            // if you inspect the fileSystem, after running this line, the 
+            var imagesDataFile = SamplesUtils.DatasetUtils.DownloadImages();
+
+            // Preview of the content of the images.tsv file
+            //
+            // imagePath    imageType
+            // tomato.bmp	tomato
+            // banana.jpg   banana
+            // hotdog.jpg   hotdog
+            // tomato.jpg   tomato
+
+            var data = mlContext.Data.CreateTextLoader(new TextLoader.Arguments()
+            {
+                Column = new[]
+                {
+                        new TextLoader.Column("ImagePath", DataKind.TX, 0),
+                        new TextLoader.Column("Name", DataKind.TX, 1),
+                    }
+            }).Read(imagesDataFile);
+
+            var imagesFolder = Path.GetDirectoryName(imagesDataFile);
+            // Image loading pipeline. 
+            var pipeline = mlContext.Transforms.LoadImages(imagesFolder, ("ImageObject", "ImagePath"))
+                          .Append(mlContext.Transforms.Resize("ImageObject",imageWidth: 100 , imageHeight: 100 ))
+                          .Append(mlContext.Transforms.ExtractPixels("Pixels", "ImageObject"));
```

Comment:
>Transforms [](start = 44, length = 10)

Considering specificity of domain, can we have Image subcatalog, like we have `ml.Transforms.Text,` can we have `ml.Transforms.Images` ?

Reply:
i might misremember, but Pete disliked the idea; i believe becuase of too many catalogs makes things hard to find; but @artidoro and @rogancarr have expressed the same thing: issue #2361

If i get another thumbs up frm @tomfinley, i am up for adding the catalog and moving the extensions there. 

---
In reply to: [253166949](https://github.com/dotnet/machinelearning/pull/2372#discussion_r253166949) [](ancestors = 253166949)